src/components/Table/Table.tsx

```diff
@@ -18,8 +18,8 @@ interface TableState {
     // activeScrollElement нужен, чтобы listner на скролл таблицы не срабатывал, когда скроллится скролл-бар и наоборот
     // в противном случае таблица будет дергаться при прокрутке
     activeScrollElement: ActiveScrollElementType;
-    columnsWidth: (number | undefined)[];
-    columnHeaderRefs: React.RefObject<HTMLTableHeaderCellElement>[];
+    columnsStyles: React.CSSProperties[];
+    columnHeaderRefs: React.RefObject<HTMLTableCellElement>[];
```

Comment:
Why narrowing down the type?

Reply:
HTMLTableHeaderCellElement is deprecated in typescript 4.5.4 definitions.